pandas/tests/indexing/test_coercion.py

```diff
@@ -613,13 +611,32 @@ def test_where_index_datetime(self, fill_val, exp_dtype):
                         pd.Timestamp('2011-01-03'),
                         pd.Timestamp('2012-01-04')])
 
-        if fill_val.tz:
-            self._assert_where_conversion(obj, cond, values, exp,
-                                          'datetime64[ns]')
-            pytest.xfail("ToDo: do not ignore timezone, must be object")
         self._assert_where_conversion(obj, cond, values, exp, exp_dtype)
-        pytest.xfail("datetime64 + datetime64 -> datetime64 must support"
-                     " scalar")
+
+    @pytest.mark.xfail(reason="ToDo: do not ignore timezone, must be object")
+    def test_where_index_datetimetz(self):
```

Comment:
This test was bizarrely structured above. It was given the wrong arguments for everything to _pass_ before it hit the xfail...

Good news is that one of the cases is passing, so I am splitting the tz-aware and tz-naive case and properly xfailing the tz-aware case (which was already failing on master and is orthogonal to this change)

Reply:
is there an issue for this case? if not can you create one and reference it here